.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -45,6 +59,9 @@ For example:
 
 - [ ] Verify that no errors appear in the JS console
 
+
+### Regression Tests to add
```

Comment:
We could add some guidance here. Something like a commented "Look for flows that may be affected by this change and add tests for those." 

Thoughts?

Reply:
Good idea! Here's my draft: "If this is a new feature, substantively changes an existing feature, or addresses an edge case, please note the testing steps that should be added to our regular QA checklist."

Reply:
Hmm I think it's better to leave this off for now since we don't have that process yet (for C+ to view existing regression tests or add to them). I do think it would be nice to be able to offload "adding new regression tests" to contributors / C+ eventually but maybe not at this moment

Reply:
Done